files/usr/share/cinnamon/cinnamon-settings/bin/ExtensionCore.py

```diff
@@ -910,7 +910,7 @@ def build_list(self, *args):
         updates_available = self.spices.get_n_updates()
         self.update_all_button.set_sensitive(updates_available)
         if updates_available > 0:
-            msg_text = _("Update all") + ' (' + ngettext("%d update available", "%d updates available", updates_available) % updates_available  + ')'
+            msg_text = _("Update all") + ' (' + gettext.ngettext("%d update available", "%d updates available", updates_available) % updates_available  + ')'
```

Comment:
I'd intuitively expect any future users of the API to want to use ngettext() directly, but maybe that is just me.